config.json

```diff
@@ -168,7 +168,8 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "logic",
```

Comment:
What is the reasoning behind adding the `logic` topic?

Reply:
In the begining one will have to apply a little logic to see that it can be solved by using the mathematical formulas of sum of squares of first n natural numbers and square of sum of first n natural numbers

Reply:
The `logic` topic is more applicable to exercises such as the [`zebra puzzle`](https://github.com/exercism/problem-specifications/blob/ae1cdbd67df28f0c4b2219237e8202f280dc8f19/exercises/zebra-puzzle/description.md). I don't think that it applies to this exercise.

Reply:
Can I make the change you suggested or the one who merges the request will ave to do so?

Reply:
@rahulagrawal97 You should make the change. Just remove the `logic` topic here.